concordium-node/src/p2p/peers.rs

```diff
@@ -51,7 +51,8 @@ impl P2PNode {
         self.get_peer_stats(Some(PeerType::Node)).into_iter().map(|stats| stats.local_id).collect()
     }
 
-    /// Measures the node's average byte throughput.
+    /// Measures the node's average byte throughput as bps i.e., bytes per
+    /// second.
     pub fn measure_throughput(&self, peer_stats: &[PeerStats]) -> (u64, u64) {
```

Comment:
Since this function can in principle fail I would recommend doing the following.

- have the function return `anyhow::Result<()>` (the output is never used, this function just modifies the state).
- in `maintanance` where the function is used do something like 
```if let Err(e) = measure_throuput(...) {
       error!("Could not measure througput: {}", e)
```
which will log the fact that this failed (and do nothing else).